packages/runtime/src/extensibility/facades/consumption/IncomingRequestsFacade.ts

```diff
@@ -23,6 +23,7 @@ import {
 } from "../../../useCases";
 
 export class IncomingRequestsFacade {
+    [key: string]: any;
```

Comment:
this was because I originally typed the method names as string in dual function accept/request tests, I changed it to a more specific type to remove this